examples/arg_vecs.clj

```diff
@@ -4,7 +4,7 @@
             [clojure.spec.alpha :as s]
             [grasp.api :as grasp :refer [grasp]]))
 
-(s/def ::args+body (s/cat :arg-vec vector? :exprs (s/+ seq?)))
+(s/def ::args+body (s/cat :arg-vec vector? :exprs (s/+ some?)))
```

Comment:
This change seems unrelated?

Reply:
I think this should even be `any?`

Reply:
It _is_ unrelated but I was too lazy to open up a second PR. Too bad I also got it wrong. Force pushed it out and doing that now.